lib/hash.c

```diff
@@ -2315,6 +2319,8 @@ grn_hash_add(grn_ctx *ctx, grn_hash *hash, const void *key,
     return GRN_ID_NIL;
   }
   if (!key || !key_size) {
+    ERR(GRN_INVALID_ARGUMENT,
+        "[grn][hash] key(%p) or key_size(%u) is invalid", key, key_size);
```

Comment:
`load/command_version/3/output_ids/array` test is fail by this modify.
We must fix above test case.

Could you confirm https://travis-ci.org/groonga/groonga/jobs/488390171#L3631?

Reply:
I've fixed.